packages/x-charts-pro/src/FunnelChart/FunnelPlot.tsx

```diff
@@ -72,49 +69,69 @@ const useAggregatedData = (gapIn: number | undefined) => {
       const xScale = xAxis[xAxisId].scale;
       const yScale = yAxis[yAxisId].scale;
 
-      const xPosition = (
-        value: number,
-        bandIndex: number,
-        stackOffset?: number,
-        useBand?: boolean,
+      const xPosition: PositionGetter = (
+        value,
+        bandIndex,
+        bandIdentifier,
```

Comment:
Maybe somthing like that, because the index exist for both linear and band, whereas the `bandIdentifier` can be `undefined` for linear scale


```suggestion
        ItemIndex,
        bandIdentifier,
```